docs/general/events.md

```diff
@@ -3,13 +3,34 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
 in the data model by `LogRecord`s.
 
+Semantically, an Event is a named occurrence at an instant in time. It simply signals that
+"this thing happened at this time" and provides additional specifics about the occurrence.
+Examples of Events might include things like uncaught exceptions, button clicks, user logout,
+network connection severed, etc.
+
+In OpenTelemetry, Events are implemented as a specific type of `LogRecord` that conforms to
+the conventions included here, and Events
+[have their own api](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md).
+The api abstracts away knowledge of `LogRecord` so that users are merely dealing with Event
+semantics.
+
+In addition to a required name, an Event may contain a _payload_, which can be either structured or unstructured.
```

Comment:
> We require the reader to make the logical leap to go and find the [AnyValue](https://github.com/open-telemetry/opentelemetry-proto/blob/ea449ae0e9b282f96ec12a09e796dbb3d390ed4f/opentelemetry/proto/common/v1/common.proto#L28) definition in the proto, which is asking a lot.

Yeah, that's kinda why I was trying to avoid it because hopefully it's abstracted away. What about something like this:

"In addition to a required name, an Event may contain a _payload_ of any type permitted by the [`LogRecord` body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body). "

Reply:
Should we add that Events also have a priority, in addition to name, optional payload and optional attributes? The Event API talks about priority, so it will help with being consistent with it if we add priority to the definition or description of what Events contain.

Reply:
Maybe, but if so probably in a follow-up PR. The intent of this PR was the help clarify that the payload of an event lives in the body of the LogRecord.